.github/workflows/ci.yml

```diff
@@ -140,7 +140,7 @@ jobs:
         env:
           PROVIDER_URL: 'http://172.15.0.4:8030'
       - name: docker logs
-        run: docker logs ocean_aquarius_1 && docker logs ocean_provider_1 && docker logs ocean_provider2_1 && docker logs ocean_computetodata_1
+        run: docker logs ocean-ipfs-1
```

Comment:
i think we need to revert this or maybe just add ipfs logs if we need them

Reply:
I'll revert how it was before